lib/PipelineC/Tracing/Runner.cpp

```diff
@@ -497,6 +497,14 @@ llvm::Error Trace::run(const revng::tracing::RunTraceOptions Options) const {
     TraceRunnerLogger.flush();
 
     CommandHandler[Command.Name](Context, Arguments, Command.Result);
+
+    // If an analysis was just run and the SaveAfterAnalyses is enabled, save
+    if ((Command.Name == "rp_manager_run_analysis"
+         || Command.Name == "rp_manager_run_analyses_list")
+        && Options.SaveAfterAnalyses) {
+      std::vector<revng::tracing::Argument> SaveArguments{ Arguments[0] };
+      CommandHandler["rp_manager_save"](Context, SaveArguments, "true");
```

Comment:
1. The following should work:

   ```c++
   CommandHandler["rp_manager_save"](Context, { Arguments[0] }, "true");
   ```
2. Does `rp_manager_save` saves everything or only the model?
   Ideally we'd just like to save the model.

Reply:
1. Ok
2. It saves everything. I've double check but we already *do* save the model after analyses, so the flag would be useful for e.g. looking at a `module.ll` pre-crash

Reply:
2. > I've double check but we already do save the model after analyses

   Are you 100% sure? I think when I was running the trace the resume directory remained empty.
   I'm not that interested in saving `module.ll` as of now. If we already save the model, this feature is not necessary. Saving containers after analysis is arbitrary and not in line with the original intent of this change.

Reply:
Sorry, brain fart, `PipelineManager` saves the model **after** an analysis is run.
We can keep things as-is or I can add another function in `PipelineC.cpp` that only saves the model.

Reply:
Added the command-line option `--save-after-every-analysis` to `PipelineManager`